includes/transformer/class-post.php

```diff
@@ -750,12 +755,6 @@ protected function get_type() {
 						break;
 				}
 				break;
-			case 'page':
-				$object_type = 'Page';
-				break;
-			default:
-				$object_type = 'Article';
-				break;
 		}
```

Comment:
I'm reading the logic as something like the following, starting at `// Default to Article.`:

```php
		// Default to Note.
		$object_type = 'Note';
		$post_type   = \get_post_type( $this->wp_object );

		if ( 'page' === $post_type ) {
			$object_type = 'Page';
		} else if ( 'post' === $post_type ) {
			$post_format = \get_post_format( $this->wp_object );

			if ( 'standard' === $post_format || '' === $post_format ) {
				$object_type = 'Article';
			}
		}
```

Is that about right?
What are circumstances where `$post_format` would be an empty string rather than `false`?

Reply:
you are right!

Reply:
changed

Reply:
@obenland I think I mixed that up with comment-types somehow, because the default type for comments was an empty string before the migration to "comment". Anyways thanks for pointing it out!

Reply:
@obenland sure, go for it! You can simply commit to my PR if you want!

Reply:
FYI: the list was initially way bigger, that's why it is a switch/case: https://github.com/Automattic/wordpress-activitypub/commit/dde469c27c4e5ab6c09e7b284374b94d14beb9fa#diff-28c56747da3496a58aa702e51b60e1b36656e0263c4c2d97d0a013a6d2f07a08R64